tests/validate/missing-engine-theia-package.json

```diff
@@ -0,0 +1,5 @@
+{
+    "engines": {
+        
+    }
+}
```

Comment:
.json files in tests/validate are missing EOL

Reply:
yes I will fix these EOL